backend/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon src/index.js",
-    "test": "jest --coverage --watch"
+    "test": "jest --coverage --watch --runInBand"
```

Comment:
```Alias: -i. Run all tests serially in the current process, rather than creating a worker pool of child processes that run tests. This can be useful for debugging.```

I've had issues with tests failing from time to time, I figured out that it might be because of them being run concurrently (e.g. one cleaned the database before the other finish it's job). This can be solved by rearranging the tests (cleaning the database before the tests instead of after) or by running them sequentially.